Performance/FlxBunnyMark/Project.xml

```diff
@@ -33,6 +33,7 @@
 	
 	<!--------------------------------LIBRARIES------------------------------------->
 
+	<set name="next"/>
```

Comment:
I'm hesistant to change the default to `next` before we have a workaround for #228. We could probably just let the background be rendered without `FlxStrip` for OpenFL Next? A `FlxBackdrop` or something?